common/tasks.py

```diff
@@ -0,0 +1,74 @@
+import time
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.mail import send_mail
+from django.utils import timezone
+
+from studio.celery import app
+from studio.utils import get_logger
+
+logger = get_logger(__name__)
+
+ADMIN_EMAIL = "serve@scilifelab.se"
+
+
+@app.task
+def handle_deleted_users():
+    """
+    Handles deleted user accounts.
+
+    Using a threshold of 12 months. If the user deleted their account longer ago, then it is handled.
+    Initially, user accounts deleted more than 12 months ago will trigger an email to Serve admins
+    for manual processing.
+
+    TODO: Make threshold_days a variable in settings.py
+    """
+
+    threshold_days = 365
+    threshold_time = timezone.now() - timezone.timedelta(days=threshold_days)
+
+    logger.info(f"Running task handle_deleted_users using threshold {threshold_days} days")
+
+    # Get all inactive users
+    inactive_users = User.objects.filter(is_active=False)
```

Comment:
You can add multiple conditions here, and remove the 
```python
       if user.userprofile.deleted_on is None:
            # This user was not deleted. It is inactive for another reason.
            continue
```


```suggestion
    inactive_users = User.objects.filter(is_active=False, userprofile__deleted_on__isnull=False)
```

but i have to try this locally on monday.